scripts/generate_coverage/merge_cobertura.py

```
@@ -0,0 +1,42 @@
+
+def main(
+    rust_file: Path = RUST_FILE_OPT,
+    python_file: Path = PYTHON_FILE_OPT,
+    output_path: Path = OUTPUT_PATH_OPT,
+) -> None:
+    """Merge two cobertura XML files and delete the inputs."""
+    try:
+        output = uvx["merge-cobertura", str(rust_file), str(python_file)]()
+    except ProcessExecutionError as exc:
+        typer.echo(
+            f"merge-cobertura failed with code {exc.retcode}: {exc.stderr}", err=True
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)